portal/src/graphql/portal/AdminAPIConfigurationScreen.tsx

```diff
@@ -261,10 +261,12 @@ const AdminAPIConfigurationScreen: React.FC =
     }
 
     return (
-      <AdminAPIConfigurationScreenContent
-        appID={appID}
-        queryResult={queryResult}
-      />
+      <div className={styles.content}>
+        <AdminAPIConfigurationScreenContent
+          appID={appID}
+          queryResult={queryResult}
+        />
+      </div>
```

Comment:
Teach ScreenContent to support `scrollable` which is false by default. For screens that do not use CommandBarContainer, make them to use `<ScreenContent scrollable={true} />`